src/UI/Component/Alert/Standard/Factory.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+/* Copyright (c) 2018 Thomas Famula <famula@leifos.de> Extended GPL, see docs/LICENSE */
+
+namespace ILIAS\UI\Component\Alert\Standard;
+
+/**
+ * This is how a factory for Standard Alerts looks like.
+ */
+interface Factory {
+	/**
+	 * ---
+	 * description:
+	 *   purpose: >
+	 *     The system failed to complete some actions and displays information about the failure (brand-danger).
```

Comment:
Please use the [less-section](https://github.com/ILIAS-eLearning/ILIAS/blob/release_5-3/src/UI/docu/rules.md#introduction-of-new-ui-components) for reference to the less-variables.

Reply:
@klees We did not find any example for this section. Where should it be placed exactly in the yaml hierarchy?

Reply:
AFAIC just on top-level.

@Amstutz Do the system styles read the `less`-section?

Reply:
@klees We were not able to make a less section appear. We move this info to composition now, since it states similar information for other components (e.g. UTF8 codes for glyphs).